next-gen/test/OpenTelemetry.AutoInstrumentation.Sdk.Tests/ProtobufOtlpMetricExporterAsyncTests.cs

```diff
@@ -42,7 +43,7 @@ public async Task ToOtlpResourceMetricsTest()
         Assert.NotNull(collectedData);
         Assert.Single(collectedData.Request.ResourceMetrics);
 
-        var resourceMetric = collectedData.Request.ResourceMetrics.First();
+        var resourceMetric = collectedData.Request.ResourceMetrics[0];
```

Comment:
Also in other places

```suggestion
        var resourceMetric = Assert.Single(collectedData.Request.ResourceMetrics);
```